datahub-web-react/src/graphql/user.graphql

```diff
@@ -10,9 +10,13 @@ query getUser($urn: String!, $groupsCount: Int!) {
             lastName
             fullName
             email
+            departmentName
         }
-        editableInfo {
+        editableProperties {
+            slack
```

Comment:
We should also fetch

displayName
title
email

From here. And then use them as the defaults! Editable information should _always_ override the "info" information on the frontend side

Reply:
sure, changes are already added earlier.